parts/windowsProvision.ps1

```diff
@@ -349,6 +355,36 @@ function Install-PSW {
     }
 }
 
+function Install-AzureDCAP{
+    $installDir = Join-Path $PACKAGES_DIRECTORY "AzureDCAP"
+    Rename-item -Path $PACKAGES["AzureDCAP"]["local_file"] -NewName $PACKAGES["AzureDCAP"]["renamed_file"]
+    Install-ZipTool $PACKAGES["AzureDCAP"]["renamed_file"] `
+                    -InstallDirectory $installDir
+    $p = Start-Process powershell -Wait -NoNewWindow -PassThru -argument "$installDir\script\InstallAzureDCAP.ps1"

```

Comment:
Where does the "InstallAzureDCAP.ps1" file exist? Should that be in this PR?

Reply:
Ohh, I see. That's in the azure.dcap.windows.0.0.1.nupkg file that gets extracted, right?

Reply:
yes, your second comment is correct. It is referring to the ps script in the extracted NuGet package.